src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -2253,6 +2253,10 @@ public override BoundNode VisitBinaryOperator(BoundBinaryOperator node)
                 Debug.Assert(!node.OperatorKind.IsUserDefined());
                 VisitBinaryLogicalOperatorChildren(node);
             }
+            else if (node.InterpolatedStringHandlerData is { } data)
```

Comment:
I'm surprised the `AbstractFlowPass` only has to deal with `InterpolatedStringHandlerData` in `VisitBinaryOperator`. Would `VisitConversion` or `VisitCall` (or `VisitArguments...`) need some handling too, like `LocalRewriter`?